src/Mod/BIM/ArchComponent.py

```diff
@@ -1406,7 +1419,10 @@ def isFaceHorizontal(self, face):
 
     def isRightAngle(self, angle):
         """Check if the angle is close to 90 degrees."""
-        return 1.57 < angle < 1.571
+        return round(angle, 3) == 1.571
```

Comment:
you can use `math.isclose` and `math.pi/2` to avoid the magic number. Same below